### ISSUE TITLE
Extension issue

### ISSUE THREAD

**Issue by Sudeep732000** [ritwickdey/vscode-live-server#2227]
- Issue Type: `Bug`
- Extension Name: `LiveServer`
- Extension Version: `5.6.1`
- OS Version: `Windows_NT x64 10.0.19043`
- VS Code version: `1.62.0`

:warning: We have written the needed data into your clipboard. Please paste! :warning:
```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 20852,
		"activateCallTime": 15,
		"activateResolvedTime": 11,
		"activationReason": {
			"startup": true,
			"extensionId": {
				"value": "ritwickdey.LiveServer",
				"_lower": "ritwickdey.liveserver"
			},
			"activationEvent": "*"
		}
	},
	"runtimeErrors": [],
	"runningLocation": 1
}
```

**Comment by Sudeep732000** [comment]
Recently i am facing this issue with my live server Extension which takes more time to load while startup or it becomes unresponsive please look into it and let me know any updates regarding this issue

**Comment by heewoonkim2020** [comment]
> Recently i am facing this issue with my live server Extension which takes more time to load while startup or it becomes unresponsive please look into it and let me know any updates regarding this issue

Yes, that happens to me too. I guess the developers are done checking GitHub now.. When will this ever get fixed?
